.swiftlint.yml

```diff
@@ -15,7 +15,6 @@ enabled_rules:
   - modifier_order
   - missing_docs
   - implicit_return
-  - function_default_parameter_at_end
```

Comment:
I removed this rule because pretty much every violation it found was justified imo - i.e. if a method has like 20 parameters I prefer to have them alphabetically ordered rather than order by default or not.